packages/flutter/test/widgets/independent_widget_layout_test.dart

```diff
@@ -96,6 +96,51 @@ class TriggerableState extends State<TriggerableWidget> {
   }
 }
 
+class TestFocusable extends StatefulWidget {
+  const TestFocusable({
+    Key key,
+    this.focusNode,
+    this.autofocus: true,
+  }) : super(key: key);
+
+  final bool autofocus;
+  final FocusNode focusNode;
+
+  @override
+  TestFocusableState createState() => new TestFocusableState();
+}
+
+class TestFocusableState extends State<TestFocusable> {
+  bool _didAutofocus = false;
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    if (!_didAutofocus && widget.autofocus) {
+      _didAutofocus = true;
+      FocusScope.of(context).autofocus(widget.focusNode);
+    }
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+  }
```

Comment:
Remove?  This function just forwards to its base class.

Reply:
Yes, thanks, I should clean this up a bit (I cloned the similar class from test_focus and removed some code, but I can remove more).